code/lib/instrumenter/package.json

```diff
@@ -54,6 +54,7 @@
   "publishConfig": {
     "access": "public"
   },
+build/lib-docs-tools+lib-instrumenter-tsup
```

Comment:
Here's another instance. Just needs to be deleted and the code is as you specified.